internal/web/controllers/users-controller.go

```diff
@@ -45,6 +45,33 @@ func (c *UsersController) NewUser(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusCreated)
 }
 
+func (c *UsersController) UpdateUserById(w http.ResponseWriter, r *http.Request) {
```

Comment:
Nesse caso não precisa especificar que é por id o update, no momento só temos esse método que faz isso para os usuários. Como vamos ter os adms depois, talvez seja interessante renomear para `UpdateClient`, mas é só uma sugestão. `UpdateUser` já deixa bem claro o propósito.
```suggestion
func (c *UsersController) UpdateUser(w http.ResponseWriter, r *http.Request) {
```